make/mt-config/mt-tls-wolfssl-esp32.mk

```diff
@@ -3,19 +3,10 @@
 # This is part of the Xively C Client codebase,
```

Comment:
👍

Reply:
I could not solve otherwise, since the mt-esp32.mk solves the compiler availability. A different make run can not benefit from it. So I had to combine these two.